lean-client-js-browser/src/inprocess.ts

```diff
@@ -136,17 +154,41 @@ export function loadJsOrWasm(urls: LeanJsUrls, loadJs: (url: string) => Promise<
     }
 }
 
-export function loadBufferFromURL(url: string): Promise<Buffer> {
-    if (!url) {
-        return null;
-    }
-    return new Promise<Buffer>((resolve, reject) => {
+function loadInfoJson(infoUrl: string): Promise<string> {
+    // if this fails let's continue (we just won't be able to resolve lean files to github)
+    return new Promise<string>((resolve, reject) => {
+        const req = new XMLHttpRequest();
+        req.responseType = 'text';
+        req.open('GET', infoUrl);
+        req.onloadend = (e) => {
+            if (req.status === 200) {
+                resolve(req.responseText);
+            } else {
+                console.log(`could not fetch ${infoUrl}: http code ${req.status} ${req.statusText}`);
+                resolve(null);
+            }
+        };
+        req.onerror = (e) => {
+            console.log(`error fetching ${infoUrl}: ${e}`);
+            resolve(null);
```

Comment:
```suggestion
            reject(e);
```
?

Reply:
Thanks, I changed this function (`loadInfoJson`) so that it actually rejects errors. The downstream functions should now handle them, if necessary. 

The info.json file is not strictly necessary if we have to fall back to the uncached path (if it's not there, the source file locations just won't be rewritten to the github URLs), so `loadBufferFromURL` is allowed to ignore errors thrown by this.